mintme/src/components/form/CollectibleForm.tsx

```diff
@@ -91,6 +92,8 @@ export const CollectibleForm = (props: FormProps) => {
     handleRedirect()
   }, [handleRedirect, isSetLoading])
 
+  useRouteChangePrompt(dirty)
```

Comment:
Does this need to get reset to false ever?

Reply:
Yes when ever we submit/redirect to signin this needs to be false